base/shared/src/main/scala/scalaz/tc/foldable.scala

```diff
@@ -30,6 +32,9 @@ object FoldableClass {
   trait DeriveFoldMap[F[_]] extends FoldableClass[F] with Alt[DeriveFoldMap[F]] {
     final override def foldMap[A, B](fa: F[A])(f: A => B)(implicit B: Monoid[B]) =
       foldRight(fa, B.mempty)((a, b) => B.mappend(f(a), b))
+
+    final override def msuml[A](fa: F[A])(implicit A: Monoid[A]) =
+      foldLeft(fa, A.mempty)((a, s) => A.mappend(a, s))
```

Comment:
Maybe `A.mappend(_, _)`? Btw if `s` here means `sum`, it should be on the left. Semantically it looks correct.

Reply:
Why on the left? I don't follow.